src/Admin/AbstractAdmin.php

```diff
@@ -812,7 +812,7 @@ public function getFilterParameters()
 
         // build the values array
         if ($this->hasRequest()) {
-            $filters = $this->request->query->get('filter', []);
+            $filters = $this->request->query->get('filter') ?: [];
```

Comment:
``` 
  4x: Since symfony/http-foundation 5.1: Passing a non-string value as 2nd argument to "Symfony\Component\HttpFoundation\InputBag::get()" is deprecated, pass a string or null instead.
    2x in CRUDControllerTest::testList from Sonata\AdminBundle\Tests\Functional\Controller
    2x in CRUDControllerTest::testEmptyList from Sonata\AdminBundle\Tests\Functional\Controller
```